src/py42/sdk/queries/fileevents/file_event_query.py

```diff
@@ -8,18 +8,18 @@
 class FileEventQuery(BaseQuery):
     """Helper class for building Code42 Forensic Search queries.
 
-    A FileEventQuery instance's ``all()`` and ``any()`` take one or more :class:`FilterGroup`
-    objects to construct a query that can be passed to the :meth:`FileEventClient.search()` method.
-    ``all()`` returns results that match all of the provided filter criteria, ``any()`` will return
-    results that match any of the filters.
+    A FileEventQuery instance's ``all()`` and ``any()`` take one or more
+    :class:`~py42.sdk.queries.query_filter.FilterGroup` objects to construct a query that
+    can be passed to the :meth:`FileEventClient.search()` method. ``all()`` returns results
```

Comment:
Note that this link does not actually work because it is referencing an internal client. Still, I like that it's there because it tells me, the developer, that it's a method at least, and in a form that I am used to.